test/quantization/test_qat.py

```diff
@@ -68,7 +68,7 @@
     TORCH_VERSION_AT_LEAST_2_5,
 )
 
-from torchao.quantization.GPTQ import (
+from torchao.quantization.quantize_linear import (
```

Comment:
this will be bc breaking since it's used by other repos like torchchat, so we'd probably want to keep backward compatibility by still keeping the old import path